lms/static/scripts/frontend_apps/components/dashboard/DashboardFooter.tsx

```diff
@@ -0,0 +1,47 @@
+import type { LinkProps } from '@hypothesis/frontend-shared';
+import { Link } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+
+function FooterLink({
+  classes,
+  ...rest
+}: Omit<LinkProps, 'unstyled' | 'target'>) {
+  return (
+    <Link
+      {...rest}
+      classes={classnames(classes, 'hover:underline')}
+      unstyled
+      target="_blank"
+    />
+  );
+}
+
+export default function DashboardFooter() {
+  return (
+    <div className="w-full bg-grey-9 text-white p-3">
+      <div className="mx-auto max-w-6xl flex justify-between items-center">
+        <div className="flex gap-3">
+          <FooterLink href="https://web.hypothes.is" classes="font-bold">
+            <img
+              alt="Hypothesis logo"
+              src="/static/images/hypothesis-logo.png"
+              className="mr-2 inline"
+            />
+            hypothes.is
+          </FooterLink>
```

Comment:
Oh! Yeah, I didn't want to add noreferrer, since we are linking to a well known page.

I guess the `Link` component automatically adds that if `target` is `_blank`. I'll take a look.

Reply:
I suspect those rules came from an ESLint check, which we could customize, so that links everywhere have a referrer enabled by default, leaving it up to the browser to truncate the referrer for security purposes.

Reply:
Some notes on this: https://stackoverflow.com/a/64777573/434243